exec/java-exec/src/test/java/org/apache/drill/exec/planner/logical/TestConvertCountToDirectScan.java

```diff
@@ -159,4 +159,30 @@ public void ensureConvertForSeveralColumns() throws Exception {
     }
   }
 
+  @Test
+  public void ensureCorrectCountWithMissingStatistics() throws Exception {
+    dirTestWatcher.copyResourceToRoot(Paths.get("parquet", "wide_string.parquet"));
+
+    test("use dfs.tmp");
+    String tableName = "wide_str_table";
+    try {
+      // table will contain two partitions: one - with null value, second - with non null value
+      test("create table %s partition by (col_str) as select * from dfs.`parquet/wide_string.parquet`", tableName);
```

Comment:
I think table `wide_string.parquet` may be accessible from `cp` workspace, so please remove line `dirTestWatcher.copyResourceToRoot()` above and change `dfs` to `cp` in the query if it will work.